playbooks/roles/custom_domains/tasks/main.yml

```
@@ -5,24 +5,33 @@
     - custom_domains
     - custom_domains:get_domains
 
-- name: "Fetch custom domains list"
+- debug:
+    var: edxapp_venv_bin
+- debug:
+    var: EDXAPP_SETTINGS
+- debug:
+    var: edxapp_code_dir
+- debug:
+    var: groups
+    
+- name: Fetch custom domains list
   become_user: "{{ edxapp_user }}"
-  django_manage: >
-    command="lms custom_domains_list"
-    app_path="{{ edxapp_code_dir }}"
-    settings="{{ EDXAPP_SETTINGS }}"
-    pythonpath="{{ edxapp_code_dir }}"
-    virtualenv="{{ edxapp_venv_dir }}"
+  shell: >
+    {{ edxapp_venv_bin }}/python manage.py lms --setting={{ EDXAPP_SETTINGS }} custom_domains_list
```

I think we need to add `$ source`

Usually it bites at the very wrong time. Having the explicit env enabled is better than just pointing to the correct binary. Same as https://github.com/edx/configuration/pull/4812

```suggestion
    . {{ edxapp_app_dir }}/edxapp_env && {{ edxapp_venv_bin }}/python manage.py lms --setting={{ EDXAPP_SETTINGS }} custom_domains_list
```